styles/globals.css

```diff
@@ -0,0 +1,332 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400;1,500;1,700&display=swap");
+
+:root {
+  --primary-green: #0d712e;
+  --primary-yellow: #ffd900;
+  --primary-blue: #022777;
+  --primary-white: #f5f5fa;
+
+  --secundary-blue: #1c3183;
+  --secundary-dark-green: #00450e;
+  --secundary-green: #0b6128;
+  --secundary-light-green: #069736;
+  --secundary-yellow: #1c3183;
+  --secundary-red-warning: #ed3233;
+
+  --tertiary-light-blue: #b0c4d5;
+  --tertiary-dark-blue: #104b7e;
+  --tertiary-yellow: #ffcb21;
+  --tertiary-background: rgba(194, 18, 18, 0.15);
+
+  --neutral-black: #1a1a1a;
+  --neutral-dark: #525252;
+  --neutral-medium-dark: #525252;
+  --neutral-medium: #9b9b9f;
+  --neutral-medium-light: #c7c8d0;
+  --neutral-light: #f2f2f7;
+  --neutral-white: #ffffff;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 32px;
+}
+
+html,
+body {
+  max-width: 100vw;
+  overflow-x: hidden;
+}
+
+.display,
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-weight: 700;
+}
+
+.display {
+  font-size: 58px;
```

Comment:
Please use rem for font sizes https://www.sitepoint.com/understanding-and-using-rem-units-in-css/

Reply:
The style guide has those values, with a min and a max. I set max to Desktop view and min to mobile view. But I'm not sure if that was the intention.

I’ll change them to rem.

Reply:
Done